src/components/ImportFromCognitoS3Dialog/index.js

```diff
@@ -48,17 +74,64 @@ const ExpandedRow = ({ data }) => {
         paginationRowsPerPageOptions={[10, 20, 25, 50, 100, 200]}
         customStyles={customStyles}
       />
+      <DataTable
+        style={{
+          boxSizing: "border-box",
+          paddingLeft: "50px",
+          paddingRight: "10px",
+        }}
+        dense
+        striped
+        noHeader
+        columns={expandedAssetsColumns}
+        data={rowAssets}
+        noDataComponent={'Make sure the project has "data" folder'}
+        pagination={rowAssets.length > 10}
+        paginationPerPage={10}
+        paginationRowsPerPageOptions={[10, 20, 25, 50, 100, 200]}
+        customStyles={customStyles}
+      />
     </>
   )
 }
 
 export default ({ open, onClose, onAddSamples }) => {
   const [dm, setDm] = useState(null)
+  const [oldData] = useDataset()
```

Comment:
you can't use this unless the dialog is never rendered when it's called, it'll crash datasets otherwise

You'll need to adjust the code that calls `<ImportFromCognitoS3Dialog />` to only conditionally render it.

Reply:
Little question : if i made sure the dialogBox is not rendered before the button click can i ignore this comment (and it is a good idea to implement this for all the dialogBox)

Reply:
yep! nice